clients/cli/src/copy.rs

```diff
@@ -4,16 +4,18 @@ use std::path::PathBuf;
 
 use lockbook_core::Core;
 use lockbook_core::CoreError;
-use lockbook_core::CreateFileAtPathError;
-use lockbook_core::DecryptedFileMetadata;
 use lockbook_core::Error as LbError;
-use lockbook_core::GetFileByPathError;
+use lockbook_core::FileType::Folder;
```

Comment:
I think we should only import the `FileType`, no specific variants.

Reply:
db4d3181